packages/components/src/table/stories/table-summary.js

```diff
@@ -2,13 +2,20 @@
  * External dependencies
  */
 import { TableSummary } from '@woocommerce/components';
+import { Card, CardFooter } from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
 import { summary } from './index';
 
-export const Basic = () => <TableSummary data={ summary } />;
+export const Basic = () => (
+	<Card>
+		<CardFooter justify="center">
+			<TableSummary data={ summary } />
+		</CardFooter>
+	</Card>
+);
```

Comment:
A bit out of scope but this helps increase the visualization in the canvas section of this component in Storybook. 

| Before  | After |
| ------------- | ------------- |
| ![before](https://user-images.githubusercontent.com/10045087/124888188-11e87180-e000-11eb-891f-85877031837b.png)  | ![after](https://user-images.githubusercontent.com/10045087/124888253-1f9df700-e000-11eb-85bf-66d854a9bbf2.png) |